Added app shortcut icon quality measure on Android (#3091)
GoogleChrome/web.dev: src/site/content/en/blog/app-shortcuts/index.md

SVG files are not supported at the time of writing, use PNG instead.

If you want pixel-perfect icons, provide them in increments of 48dp (i.e. 36x36,
48x48, 72x72, 96x96, 144x144, 192x192 pixel icons). Otherwise, it is recommended
that you use a single 192x192 pixel icon.

> As a quality measure, icons must be at least half of the device's ideal size
> (48dp) on Android. This means the shortcut icon won't be visible on

Reviewer: ```suggestion
As a quality measure, icons must be at least half of the device's ideal size,
which is 48dp on Android. This means the shortcut icon won't be visible on
```

> As a quality measure, icons must be at least half of the device's ideal size,
> which is 48dp on Android. This means the shortcut icon won't be visible on

Reviewer: ```suggestion
As a quality measure, icons must be at least half of the device's ideal size on
Android, which is 48dp. This means the shortcut icon won't be visible on
```

> Android, which is 48dp. This means the shortcut icon won't be visible on
> [xxhdpi screens] for instance if a 72x72 pixel icon is not included (`48 x (480
> / 160) / 2 = 72`).

Reviewer: ```suggestion
Android, which is 48dp. For example, to display on an [xxhdpi screen], the icon must be at least 72 by 72 pixels. (This is derived from the [formula for converting] dp units for pixel units.)
```

Reviewer: https://developer.android.com/training/multiscreen/screendensities#dips-pels

SVG files are not supported at the time of writing, use PNG instead.

If you want pixel-perfect icons, provide them in increments of 48dp. Otherwise,

> it is recommended that you use a 192x192 pixels icon.

Reviewer: ```suggestion
it is recommended that you use a 192x192 pixel icon.
```

{% Aside %}
As a quality measure, icons must be at least half of the device's ideal size
(48dp) on Android. This means the shortcut icon won't be visible on xxhdpi
screens for instance if a 72x72 pixels icon is not included (`48 x (480 / 160) /

> 2 = 72`).

Reviewer: I don't understand this sentence.

Author: Which part?

Author: Shall I link to an article explaining xxhdpi displays?

Reviewer: The icon being invisible if it's too small seems like part of the main point, but you've attached it to the example. If 48 is the device's ideal, why does 72x72 produce no result?

Author: Dp and pixels are two different things.
